.travis.yml

```
@@ -0,0 +1,30 @@
+language: node_js
+cache: npm
+stages:
+  - check
+  - test
+  - cov
+
+node_js:
+  - '10'
+
+os:
+  - linux
+  - osx
+
+script: npx nyc -s npm run test:node -- --bail
+after_success: npx nyc report --reporter=text-lcov > coverage.lcov && npx codecov
+
+jobs:
+  include:
+    - os: windows
+      cache: false
+
+    - stage: check
+      script:
+        - npx aegir commitlint --travis
+        - npx aegir dep-check
+        - npm run lint
```

Format might be off on this, but we should add the browser and webworker builds too.

```suggestion
        - npm run lint

    - stage: test
       name: chrome
       addons:
         chrome: stable
       script:
         - npx aegir test -t browser
         - npx aegir test -t webworker

    - stage: test
       name: firefox
       addons:
         firefox: latest
       script:
         - npx aegir test -t browser -- --browsers FirefoxHeadless
         - npx aegir test -t webworker -- --browsers FirefoxHeadless    
```